packages/extension-ui/src/Popup/ImportSeed.tsx

```diff
@@ -2,39 +2,63 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import React, { useContext, useState } from 'react';
+import React, { useContext, useRef, useState } from 'react';
 
 import {
   ActionContext,
   Address,
   Button,
   ButtonArea,
   Header,
-  TextAreaWithLabel,
-  VerticalSpace
+  TextAreaWithLabel
 } from '../components';
 import { createAccountSuri, validateSeed } from '../messaging';
-import { Name, Password } from '../partials';
+import { Name, Password, DerivationPath } from '../partials';
 import styled from 'styled-components';
+import { OnDerivationPathChangeProps } from '@polkadot/extension-ui/partials/DerivationPath';
+import { KeypairType } from '@polkadot/util-crypto/types';
```

Comment:
Move these to below the react import. Always -

- import from names packages
- <blank line>
- import from relative paths

(alphabetical in these sections)